code-challenges.test.js

```diff
@@ -14,6 +14,23 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
+describe("capitalName", () => {
```

Comment:
I am considering this section as only complete since there is no code logic attempted in the functions.  While pseudocode helps walk through the logic and is essential to creating a roadmap, it will not help you work through errors or get functioning logic until the code is actually written.